File: doc/how-tos/save-schema-and-cred-def/python/step4.py
        # 11.
        print_log('\n11. Creating and storing CRED DEFINITION using anoncreds as Trust Anchor, for the given Schema\n')
        cred_def = await anoncreds.issuer_create_and_store_claim_def(
            wallet_handle, trust_anchor_did, json.dumps(schema), 'CL', False)
        print_log('Returned Cred Definition:\n')
        pprint.pprint(json.loads(cred_def))

        # 12.
        print_log('\n12. Closing wallet and pool\n')
        await wallet.close_wallet(wallet_handle)
        await pool.close_pool_ledger(pool_handle)

        # 13.
        print_log('\n13. Deleting created wallet\n')
        await wallet.delete_wallet(wallet_name, None)

        # 14.
        print_log('\n14. Deleting pool ledger config\n')
        await pool.delete_pool_ledger_config(pool_name)

File: doc/how-tos/save-schema-and-cred-def/java/step4.java
		System.out.println("\n11. Creating and storing CRED DEF using anoncreds as Trust Anchor, for the given Schema\n");
		String credDefJSON = "{\"seqNo\": 1, \"dest\": \"" + defaultStewardDid + "\", \"data\": " + schemaDataJSON + "}";
		System.out.println("Cred Def JSON:\n" + credDefJSON);
		String credDef = issuerCreateAndStoreClaimDef(walletHandle, trustAnchorDID, credDefJSON, "CL", false).get();
		System.out.println("Returned Cred Definition:\n" + credDef);

		// Some cleanup code.
		System.out.println("\n12. Close and delete wallet\n");
		walletHandle.closeWallet().get();
		Wallet.deleteWallet(walletName, null).get();

		System.out.println("\n13. Close pool\n");
		pool.closePoolLedger().get();

		System.out.println("\n14. Delete pool ledger config\n");
		Pool.deletePoolLedgerConfig(poolName).get();

